client/homebrew/pages/userPage/userPage.jsx

```diff
@@ -31,9 +31,10 @@ const UserPage = createClass({
 
 		const brewCollection = [
 			{
-				title : `${usernameWithS} published brews`,
-				class : 'published',
-				brews : brews.published
+				title   : `${usernameWithS} published brews`,
+				class   : 'published',
+				brews   : brews.published,
+				visible : true
```

Comment:
Can we remove this `visible : true` as well?

Reply:
Should be removed in the latest commit.